packages/rich-text/src/component/index.js

```diff
@@ -949,11 +929,7 @@ class RichText extends Component {
 	 * @return {Object} An internal rich-text value.
 	 */
 	formatToValue( value ) {
-		const {
-			format,
-			__unstableMultilineTag: multilineTag,
-			preserveWhiteSpace,
-		} = this.props;
+		const { format, __unstableMultilineTag: multilineTag, preserveWhiteSpace } = this.props;
```

Comment:
@epiqueras @jsnajdr Here's an example.

Reply:
Yeah, that fits under our line-length rule.

Reply:
What's the line length rule? I don't see any previous mention of a line length rule. I thought we set it to maximum 80.

Reply:
100 for WP.

Reply:
The current Prettier config sets the maximum to 100:

https://github.com/WordPress/gutenberg/blob/f63053cace3c02e284f00918e1854284c85b9132/packages/scripts/config/.prettierrc.js#L4

Reply:
Why 100? Until now we've formatted docs with 80 (at least that's what I've been doing) and I format code the same way. Prettier even [recommends 80](https://prettier.io/docs/en/options.html). Cc @aduth since I think you've formatted with 80 too.

Reply:
> Why 100? Until now we've formatted docs with 80 (at least that's what I've been doing) and I format code the same way.

I think that's a mistake that went uncaught in the Prettier PR. Let's change the line length to 80.

Reply:
The standard is not strict, but recommended at 80-100:

>Lines should usually be no longer than 80 characters, and should not exceed 100 (counting tabs as 4 spaces). This is a “soft” rule, but long lines generally indicate unreadable or disorganized code.

https://developer.wordpress.org/coding-standards/wordpress-coding-standards/javascript/#spacing

I personally use an [`editor.rulers` VSCode setting](https://code.visualstudio.com/docs/getstarted/settings#_default-settings) to show a line at 80th column and adhere to it quite strictly in my own code. I've been struggling with this a bit more recently, at least with mixing more complex types into JSDoc ([example of me relenting](https://github.com/WordPress/gutenberg/pull/19856/files#diff-106913d5079b57c6032c0775292d8d9dR66-R77)). For me, it's more of a mechanical, imperfect means to try to encourage myself flatten/simplify code. The difference between 80, 100, or maybe even 120 isn't going to make a huge difference in this. Unfortunately, at least for my purposes, I think it's one of Prettier's objectives to push this out of mind altogether (for better or worse).

I suppose we can still expect Prettier to collapse some of these destructurings even at 80 line length, but it is what it is.

Reply:
It's a bit strange to me that we are now enforcing 80-100 lines on a single line. So currently it's wrong to destructure at the 80th column. The rule is strict.

I do think it's sometimes a bit annoying for docs (if types and variables are long), but for code it's almost never a difficulty to fit it within 80.

Reply:
Docs can be up to 100, though not sure if Prettier will take this into account:
> https://developer.wordpress.org/coding-standards/inline-documentation-standards/javascript/#line-wrapping
> DocBlock text should wrap to the next line after 80 characters of text. If the DocBlock itself is indented on the left 20 character positions, the wrap could occur at position 100, but should not extend beyond a total of 120 characters wide.

Reply:
It won't: https://prettier.io/docs/en/options.html#print-width.

That's fine.

Reply:
Prettier doesn't actually touch comments, it leaves them as-is

See https://prettier.io/docs/en/rationale.html#comments
> When it comes to the contents of comments, Prettier can’t do much really. Comments can contain everything from prose to commented out code and ASCII diagrams. Since they can contain anything, Prettier can’t know how to format or wrap them. So they are left as-is. The only exception to this are JSDoc-style comments (block comments where every line starts with a *), which Prettier can fix the indentation of.